gnome-shell/src/gnome-shell-sass/_common.scss

```diff
@@ -878,19 +878,15 @@ StScrollBar {
 
     .system-status-icon,
     .app-menu-icon > StIcon,
-    .popup-menu-arrow {
-      icon-shadow: 0px 1px 4px rgba(0, 0, 0, 0.6);
-    }
+    .popup-menu-arrow { }
```

Comment:
I guess you can probably remove this definition at all, no?

Reply:
I left it as a kind of placeholder, since the transparency will probably return, but you are right, there is no real need.